src/stravalib/model.py

```diff
@@ -904,13 +951,18 @@ class Activity(
     """
 
     # field overrides from superclass for type extensions:
```

Comment:
i didn't check any of these objects - so i left TODO's as a reminder that we should check them.

Reply:
i think i'd prefer to check the test of this in a separate PR if you are ok with that. this pr feels bigger than i wanted it to be.